### DISCUSSION TITLE
inconsistency between fields.json implied xml node structure and xml example (OPT-300-Procedure-Buyer)

### DISCUSSION THREAD

**Discussion by karel-debrabandere-bosa-fgov-be** [OP-TED/eForms-SDK#171] [Q&A] [answered]
Hi,

The OPT-300-Procedure-Buyer field definition has ND-Root as its parentNodeId. The xpathAbsolute and xpathRelative attributes match this.
In one of the provides xml examples, however, this BT seems to be included into a repeatable <cac:ContractingParty>.

Should the parentNodeId be "ND-ContractingParty" instead?
Should the xpathRelative be "cac:Party/cac:PartyIdentification/cbc:ID" instead?

**Comment by rouschr** [comment]
@YvesJo any idea?

> In one of the provides xml examples

Which example?

**Reply by karel-debrabandere-bosa-fgov-be** [reply]
https://github.com/OP-TED/eForms-SDK/blob/develop/examples/notices/can_24_maximal.xml

**Comment by YvesJo** [comment, marked as answer]
Hi,
The whole path is actually /*/cac:ContractingParty/cac:Party/cac:PartyIdentification/cbc:ID
As there may be more than a Buyer, and only one may be referred inside cac:ContractingParty, then cac:ContractingParty has to be repeated as many times as required.
Use of metadata for rules and UI may differ sometimes and require two different path decompositions.
Such issues have been addressed and will be made available with the next release
